tests/test-bad-content-dsize-rule-2/test.yaml

```diff
@@ -14,7 +14,6 @@ checks:
         engine.message: "signature can't match as required content length 30 exceeds dsize value 10"
 
   - filter:
-      min-version: 7
```

Comment:
@inashivb should the `run.py` warn when a filter version requirement is strictly broader than the generic test version requirement ?